R/course.R

```diff
@@ -50,14 +53,15 @@ NULL
 #' @export
 use_course <- function(url, destdir = NULL) {
   url <- normalize_url(url)
-  destdir_not_specified <- is.null(destdir)
+  destdir_not_specified <- is.null(destdir) && is.null(getOption("usethis.destdir"))
```

Comment:
I think you can simplify the logic by changing the `destdir` default to `getOption("usethis.destdir")`